app.plugin.js

```diff
@@ -0,0 +1 @@
+module.exports = require("./plugin/build/withReactNativeFbads");
```

Comment:
is this file how Expo discovers plugins?

Reply:
> is this file how Expo discovers plugins?

Yes. Expo will looking for app.plugin.js to load the plugin.